src/containers/Tenant/Diagnostics/Overview/TableInfo/TableInfo.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {useMemo} from 'react';
+import cn from 'bem-cn-lite';
+
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+
+import {InfoViewer} from '../../../../../components/InfoViewer';
+
+import {getEntityName} from '../../../utils';
+
+import {prepareTableInfo} from './prepareTableInfo';
+
+import './TableInfo.scss';
+
+const b = cn('ydb-diagnostics-table-info');
+
+interface TableInfoProps {
+    data?: TEvDescribeSchemeResult;
+}
+
+export const TableInfo = ({data}: TableInfoProps) => {
+    const entityName = getEntityName(data?.PathDescription);
+
+    const {
+        generalTableInfo = [],
+        tableStatsInfo = [],
+        tabletMetricsInfo = [],
+        partitionConfigInfo = [],
+    } = useMemo(() => prepareTableInfo(data), [data]);
+
+    if (
+        [generalTableInfo, tabletMetricsInfo, partitionConfigInfo, tableStatsInfo.flat()].flat()
+            .length === 0
+    ) {
+        return <div className={b('title')}>{entityName}</div>;
+    }
+
+    return (
+        <div className={b()}>
+            <InfoViewer info={generalTableInfo} title={entityName} className={b('info-block')} />
+            <div className={b('row')}>
+                {tabletMetricsInfo.length > 0 || partitionConfigInfo.length > 0 ? (
+                    <div className={b('col')}>
+                        <InfoViewer
+                            info={tabletMetricsInfo}
+                            title={'Tablet Metrics'}
```

Comment:
Shouldn't we use i18n throughout refactored components?

Reply:
I doubted whether to use i18n for titles or not because in some places titles contain entity names that currently have mapping only in English. So it will be the mix of Russian and English. Now I added translations, It’s better to have them at least somewhere